policyengine_us/variables/household/income/person/fsla_overtime_premium.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class fsla_overtime_premium(Variable):
+    value_type = float
+    entity = Person
+    label = "premium income from overtime hours worked"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "is_eligible_for_fsla_overtime_protection"
+
+    # This variable only exists for the purpose of the tax_exempt_reform
+
+    def formula(person, period, parameters):
```

Comment:
```suggestion
    def formula_2024(person, period, parameters):
```

I see the parameters are only defined for 2024, which means they'll raise an error if we calculate over 2023. Adding this to the formula makes it return 0 for earlier years without breaking.

Reply:
No reason- but the overtime premium is added to the net income tree at any point it'll break things so this is more just for safety